### ISSUE TITLE
customize deletion_policy

### ISSUE THREAD

**Issue by lucas-seismic** [terraform-google-modules/terraform-google-sql-db#403]
### TL;DR

https://github.com/hashicorp/terraform-provider-google-beta/pull/4972
```
# google_sql_database.database will be updated in-place
  ~ resource "google_sql_database" "database" {
      ~ deletion_policy = "ABANDON" -> "DELETE"
        id              = "projects/graphite-test-rileykarson/instances/myinstance/databases/mydb"
        name            = "mydb"
        # (5 unchanged attributes hidden)
    }
```
The deletion_policy change has 2 implicit steps as follows:
1. ALTER DATABASE [library-main] SET single WITH NO_WAIT.
2. ALTER DATABASE [library-main] SET multi_user WITH NO_WAIT.

In our production, there is an exception after the first step and the database access mode wasn't changed back, so all of the services can't connect to the DB anymore. 

Could we expose the deletion_policy as a variable in this module? so we can give it a stable value

**Comment by ibhaskar2** [comment]
This issue has been fixed in https://github.com/terraform-google-modules/terraform-google-sql-db/pull/395

**Comment by saez0pub** [comment]
Hello,

I'm also facing this behaviour 👍 
It seems solved by https://github.com/terraform-google-modules/terraform-google-sql-db/commit/8ab6e374dd74a8c3f5d6963640f9eb945cd996f5 but not released
Is a released planned soon ?

**Comment by g-awmalik** [comment]
Yes this seems to have been address by #386 and should be available with v14.0. Keep an eye on this: #380. Closing this now.
